File: WebApp/Scripts/app/model/AB0053_3.js
Ext.define('WEBAPP.model.AB0053_3', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'MMCODE', type: 'string' }, //院內碼(畫面)
        { name: 'MMNAME_C', type: 'string' }, //中文品名(畫面)
        { name: 'MMNAME_E', type: 'string' }, //英文品名(畫面)
        { name: 'EXP_DATE', type: 'string' }, //月份(畫面)
        { name: 'WARNYM', type: 'string' }, //警示效期(年/月)(畫面)
        { name: 'LOT_NO', type: 'string' }, //藥品批號(畫面)
        { name: 'EXP_QTY', type: 'string' }, //數量(畫面)
        { name: 'MEMO', type: 'string' }, //備註(畫面)
        { name: 'CLOSEFLAG', type: 'string' }, //結案否
        { name: 'CREATE_TIME', type: 'string' }, //建立日期
        { name: 'CREATE_USER', type: 'string' }, //建立人員
        { name: 'UPDATE_IP', type: 'string' }, //異動IP
        { name: 'UPDATE_TIME', type: 'string' }, //異動日期
        { name: 'UPDATE_USER', type: 'string' }, //異動人員
        { name: 'MMCODE_DISPLAY', type: 'string' }, //院內碼(白底顯示)
        { name: 'EXP_DATE_DISPLAY', type: 'string' }, //有效日期(白底顯示)
        { name: 'LOT_NO_DISPLAY', type: 'string' }, //月份(白底顯示)
        { name: 'WARNYM_TEXT', type: 'string' }, //警示效期(年/月)(紅底顯示)
        { name: 'CLOSEFLAG_NAME', type: 'string' }, //結案否(畫面)
        { name: 'CLOSEFLAG_TEXT', type: 'string' }, //結案否(紅底顯示)
        { name: 'AGEN_NAMEC', type: 'string' }, //廠商
        { name: 'comb_AGEN', type: 'string' }, //廠商
        { name: 'MAIL_STATUS', type: 'string' }, //MAIL狀態
        { name: 'WARNYM_KEY', type: 'string' }, //警示效期_KEY
    ]
});

File: WebApp/Models/AB/AB0053_3.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class AB0053_3 : JCLib.Mvc.BaseModel
    {
        public string MMCODE { get; set; } //院內碼(畫面)
        public string MMNAME_C { get; set; } //中文品名(畫面)
        public string MMNAME_E { get; set; } //英文品名(畫面)
        public string EXP_DATE { get; set; } //月份(畫面)
        public string WARNYM { get; set; } //警示效期(年/月)(畫面)
        public string LOT_NO { get; set; } //藥品批號(畫面)
        public string EXP_QTY { get; set; } //數量(畫面)
        public string MEMO { get; set; } //備註(畫面)
        public string CLOSEFLAG { get; set; } //結案否
        public string CREATE_TIME { get; set; } //建立日期
        public string CREATE_USER { get; set; } //建立人員
        public string UPDATE_IP { get; set; } //異動IP
        public string UPDATE_TIME { get; set; } //異動日期
        public string UPDATE_USER { get; set; } //異動人員
        public string MMCODE_DISPLAY { get; set; } //院內碼(白底顯示)
        public string EXP_DATE_DISPLAY { get; set; } //月份(白底顯示)
        public string LOT_NO_DISPLAY { get; set; } //藥品批號(白底顯示)
        public string WARNYM_TEXT { get; set; } //警示效期(年/月)(紅底顯示)
        public string CLOSEFLAG_NAME { get; set; } //結案否(畫面)
        public string CLOSEFLAG_TEXT { get; set; } //結案否(紅底顯示)
        public string AGEN_NAMEC { get; set; } //廠商
        public string comb_AGEN { get; set; }
        public string MAIL_STATUS { get; set; } //MAIL狀態

        public string MAIL_STATUS_CODE { get; set; }    //mail狀態 原始資料
        public string EMAIL { get; set; }   // 廠商EMAIL
        public string IS_AGENNO { get; set; }   // 廠商是否變更 N-未變更 H-HIS過去資料 P-廠商檔所選資料

        public string WH_EXPQTY { get; set; }

        public string WARNYM_KEY { get; set; }
        public string AGEN_NO { get; set; }
    }
}
